### PR TITLE
Nemo/Fixed cancellation modal for all pages and redirected to Success page.

### PR DESCRIPTION
**Pull request by nemodjuric**
Connects to #1318 
mismatch

### CODE REVIEW

File: client/app/certification/CancelCertificationModal.jsx [hunk 1]
```diff
@@ -117,8 +117,7 @@ export default class CancelCertificationModal extends BaseForm {
 
     return ApiUtil.post('/certification_cancellations', { data }).
       then(() => {
-        this.props.closeHandler();
-
+        window.location.href = '/certification_cancellations/show';
```
Comment (amprokop) [inline client/app/certification/CancelCertificationModal.jsx]: hmm. does this break you out of the React app? how does it work if you navigate back or forward? we should probably make this a React component and navigate to it using React-Router.

Comment (nemodjuric) [inline client/app/certification/CancelCertificationModal.jsx]: It does "go out" of React app.However it's the final stage of certifying appeal and after this step props seem irrelevant. So any further step after Cancellation for a user would be a new task (and new set of props). Although, we can create a React component to be consistent and to leave options open for further development, especially considering minimal effort to do it.

Comment (nemodjuric) [inline client/app/certification/CancelCertificationModal.jsx]: #1722

**Comment by nemodjuric** [comment]
Here's the screenshot of success page (there is also in initial post at the very end of the gif). 
caseflow certification 11

File: client/app/certification/CancelCertificationModal.jsx (hunk 1 above)
Comment (amprokop) [inline client/app/certification/CancelCertificationModal.jsx]: cool that works.

**Comment by amprokop** [comment]
great! thanks
